packages/core/less/autocomplete.less

```diff
@@ -0,0 +1,44 @@
+@import '~react-bootstrap-typeahead/css/Typeahead.css';
+
+div.rbt-token.rbt-token-removeable {
+  background-color: @color-pf-black-150;
+  height: 20px;
+  font-size: 12px;
+  color: #363636;
```

Comment:
This should be a color from the patternfly color palette using a `color-pf-` variable, like color-pf-black-900

Reply:
In the css review they ask specifically for this color, 
though I think it can be replaced with `color-pf-black-800`

Reply:
Ah, ok. Visual designer comments override mine here 😉